liquidctl/driver/kraken3.py

```diff
@@ -1,62 +1,74 @@
-"""liquidctl drivers for fourth-generation NZXT Kraken X and Z liquid coolers.
+"""liquidctl drivers for fourth-generation NZXT Kraken X liquid coolers.
 
 Supported devices:
 
-- NZXT Kraken X (X53, X63 and Z73)
-- NZXT Kraken Z (Z53, Z63 and Z73); no LCD screen control yet
+- NZXT Kraken X (X53, X63 and X73)
+- NZXT Kraken Z (Z53, Z63 and Z73)
 
-Copyright (C) 2020–2022  Tom Frey, Jonas Malaco and contributors
+Copyright (C) 2020–2022  Tom Frey, Jonas Malaco, Shady Nawara and contributors
 SPDX-License-Identifier: GPL-3.0-or-later
 """
+# uses the psf/black style
 
 import itertools
+import io
+import math
 import logging
+import sys
 
-from liquidctl.driver.usb import UsbHidDriver
-from liquidctl.util import normalize_profile, interpolate_profile, clamp, \
-                           Hue2Accessory, HUE2_MAX_ACCESSORIES_IN_CHANNEL, \
-                           map_direction
+from PIL import Image, ImageSequence
+
+if sys.platform in ["win32", "cygwin"]:
+    from winusbcdc import *
```

Comment:
This will import a bunch of things into the namespace.  Are all of them necessary?

---

On a separate note, WinUSBCDC will raise an `ImportError` if `os.name != 'nt'`,[^1] and in a Cygwin environment `os.name == 'posix'`.[^2]

[^1]: https://gitlab.com/alelec/winusbcdc/-/blob/aa242e491822d4b371266366f175b94fac55af74/winusbcdc/__init__.py#L7
[^2]: I haven't checked it myself, but: https://stackoverflow.com/a/32612402